src/pages/ServiceList/ItemDescription.tsx

```diff
@@ -57,6 +60,14 @@ export default class ItemDescription extends React.PureComponent<Props, State> {
         <Col xs={12} sm={12} md={4} lg={4}>
           {!this.props.item.istioSidecar && <MissingSidecar />}
         </Col>
+        <Col xs={12} sm={12} md={4} lg={4}>
+          <strong>Config: </strong>{' '}
+          <ConfigIndicator
+            id={this.props.item.name + '-config-validation'}
+            validations={[this.state.validation]}
```

Comment:
Won't we always have {} values? I guess that having the following would be enough to fix this:
```typescript
validations={[this.props.item.validation]}
```
And the state could be out of the state.
Does this make sense?

Reply:
Yeah, it was supposed to be that this.props.item.validation and no state at all, that was just the UI testing code.